ui/src/components/Publication/SimpleResult/index.tsx

```diff
@@ -19,142 +19,169 @@ const SimpleResult: React.FC<Props> = (props): React.ReactElement => {
     const router = useRouter();
     const [error, setError] = React.useState('');
     const [loading, setLoading] = React.useState(false);
-    const status = useMemo(
-        () => Helpers.getPublicationStatusByAuthor(props.publicationVersion, props.user),
-        [props.publicationVersion, props.user]
-    );
+
+    const latestLiveVersion = props.publication.versions.find((version) => version.isLatestLiveVersion);
+    const authorOnLatestLive =
+        latestLiveVersion?.coAuthors.find((coAuthor) => coAuthor.linkedUser === props.user.id) ||
+        latestLiveVersion?.createdBy === props.user.id;
+    const latestVersion = props.publication.versions.find((version) => version.isLatestVersion);
+    const draftExistsWithPermission =
+        latestVersion?.currentStatus === 'DRAFT' || latestVersion?.currentStatus === 'LOCKED';
+    const draftVersion = draftExistsWithPermission ? latestVersion : false;
```

Comment:
Using `latestVersion : false` can be misleading. I guess we can replace `false` with `null` or `undefined` if latestVersion does not exists.

Reply:
Changed this to use null instead of false.